package.json

```diff
@@ -77,11 +78,12 @@
     "immutable": "3.8.2",
     "intl-messageformat": "2.2.0",
     "jquery": "=3.2.1",
+    "lodash": "4.17.11",
     "moment": "2.22.2",
     "moment-duration-format": "2.2.2",
-    "patternfly": "3.53.1",
+    "patternfly": "3.58.0",
     "patternfly-bootstrap-combobox": "1.1.7",
-    "patternfly-react": "2.11.1",
+    "patternfly-react": "2.29.15",
```

Comment:
You can remove that first commit (https://github.com/oVirt/ovirt-web-ui/pull/968/commits/9d25eaa8618170c21933e3957baa855fd029447c)